react-app/src/components/ConfigEditor/ConfigEditor.js

```diff
@@ -93,6 +131,12 @@ function ConfigEditor() {
               schema={configSchema}
               closeForm={closeForm}
             />
+            {showErrorAlert && (
+              <Alert variant="info" show={showErrorAlert} onClose={() => setShowErrorAlert(false)} dismissible>
+                <Alert.Heading>Created config from template</Alert.Heading>
+                <p>Extractors have been added for you, but other fields will still need to be filled out</p>
```

Comment:
I think this message kind of implies that the extractors themselves are totally filled out, and that the common extractor args are the only things that need to be added. I would change the message to something along the lines of `Extractors have been added for you, but CSV file paths and other fields will still need to be added`.